src/Aquifer.Common/Jobs/Queues.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Aquifer.Common.Jobs;
+
+public static class Queues
+{
+    // email
+    public const string SendEmail = "send-email";
+    public const string SendTemplatedEmail = "send-templated-email";
+
+    // notifications
+    public const string SendProjectStartedNotification = "send-project-started-notification";
+    public const string SendResourceCommentCreatedNotification = "send-resource-comment-created-notification";
+
+    // resource tracking
+    public const string TrackResourceContentRequest = "track-resource-content-request";
```

Comment:
This is a breaking change because the live queue currently has `dev` or `prod` in the name.  I'll double check that there are no abandoned queue messages after deployment and will manually replay them on the correct queue if needed.